include/language.php

```diff
@@ -335,7 +335,7 @@ public function __get( $property ) {
 		}
 
 		if ( 'search_url' === $property || 'home_url' === $property ) {
-			$url_getter = "get_{$property}";
+			$url_getter = "get_{$property}()";
```

Comment:
Won't this call `get_home_url()()` in line 342?

Reply:
Fixed in [843004c](https://github.com/polylang/polylang/pull/1219/commits/843004c3c3dc2998c3ebfa86274a20935ce9ff50).